src/components/noMatch/NoMatch.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
```

Comment:
name this file 404.jsx and no need to put it in another directory, components/404.jsx

Reply:
I will fix it in the next PR for the Routes